File: xmltojson.java
package event.xml2json;

import org.w3c.dom.*;
import java.util.*;
import javax.xml.parsers.*;

public class XmlJson extends Object {

    private static Map<String, Object> XmlToJson(Node doc) {
        Integer i;
        Map<String, Object> result;

        result = new HashMap<>();

        if ((doc != null)) {
            if ((doc.hasChildNodes())) {
                for (i = 0; i != doc.getChildNodes().getLength(); i += 1) {
                    if ((doc.getChildNodes().item(i).getNodeType() == Node.ELEMENT_NODE)) {
                        if ((!(result.containsKey("" + doc.getChildNodes().item(i).getLocalName())))) {
                            result.put("" + doc.getChildNodes().item(i).getLocalName(), new ArrayList<>());
                        }
                        if ((doc.getChildNodes().item(i) != null)) {
                            ((List) (result.get("" + doc.getChildNodes().item(i).getLocalName()))).add(XmlJson.XmlToJson(doc.getChildNodes().item(i)));
                        }
                    }
                }
            }
            if ((doc.getNodeType() == Node.ELEMENT_NODE)) {
                if ((!(result.containsKey("#" + "name")))) {
                    result.put("#" + "name", "");
                }
                if ((doc.getLocalName() != null)) {
                    result.replace("#" + "name", doc.getLocalName().trim());
                }
                if ((!(result.containsKey("#" + "text")))) {
                    result.put("#" + "text", "");
                }
                if ((doc.getFirstChild() != null)) {
                    result.replace("#" + "text", doc.getFirstChild().getNodeValue().trim());
                }
                if ((!(result.containsKey("#" + "xmlns")))) {
                    result.put("#" + "xmlns", "");
                }
                if ((doc.getNamespaceURI() != null)) {
                    result.replace("#" + "xmlns", doc.getNamespaceURI().trim());
                }
            }
            if ((doc.getNodeType() == Node.ATTRIBUTE_NODE)) {
                if ((!(result.containsKey("#" + "name")))) {
                    result.put("#" + "name", "");
                }
                if ((doc.getLocalName() != null)) {
                    result.replace("#" + "name", doc.getLocalName().trim());
                }
                if ((!(result.containsKey("#" + "text")))) {
                    result.put("#" + "text", "");
                }
                if ((doc.getNodeValue() != null)) {
                    result.replace("#" + "text", doc.getNodeValue().trim());
                }
                if ((!(result.containsKey("#" + "xmlns")))) {
                    result.put("#" + "xmlns", "");
                }
                if ((doc.getNamespaceURI() != null)) {
                    result.replace("#" + "xmlns", doc.getNamespaceURI().trim());
                }
            }
            if ((doc.hasAttributes())) {
                for (i = 0; i != doc.getAttributes().getLength(); i += 1) {
                    if ((doc.getAttributes().item(i).getNodeType() == Node.ATTRIBUTE_NODE)) {
                        if ((!(result.containsKey("@" + doc.getAttributes().item(i).getLocalName())))) {
                            result.put("@" + doc.getAttributes().item(i).getLocalName(), new HashMap<>());
                        }
                        if ((doc.getAttributes().item(i) != null)) {
                            ((Map) (result.get("@" + doc.getAttributes().item(i).getLocalName()))).putAll(XmlJson.XmlToJson(doc.getAttributes().item(i)));
                        }
                    }
                }
            }
        }

        return (result);
    }

    public static void main(String[] args) throws Throwable {
        DocumentBuilderFactory fuck;
        fuck = DocumentBuilderFactory.newInstance();
        fuck.setNamespaceAware(true);

        System.out.println(((Map) (((Map) (((List) (((Map) (((List) (XmlJson.XmlToJson(fuck.newDocumentBuilder().parse("d:/export.xml").getDocumentElement()).get("files"))).get(0))).get("file"))).get(0))).get("@name"))).get("#text"));

        return;
    }
}

File: xmltojson.py
import lxml.etree as xml

def parse(element):
    result = dict()

    if((type(element) != type(tuple()))):
        for item in element.items():
            if((not(result.has_key("@" + xml.QName(item[0]).text)))):
                result["@" + xml.QName(item[0]).text] = list()
            if((type(item) != type(None))):
                result["@" + xml.QName(item[0]).text].append(parse(item))
    if((type(element) != type(None))):
        if((type(element) != type(tuple()))):
            if((not(result.has_key("%" + "name")))):
                result["%" + "name"] = str()
            if((type(xml.QName(element.tag).localname) != type(None))):
                result["%" + "name"] += str(xml.QName(element.tag).localname).strip()
            if((not(result.has_key("%" + "text")))):
                result["%" + "text"] = str()
            if((type(element.text) != type(None))):
                result["%" + "text"] += str(element.text).strip()
            if((type(element.tail) != type(None))):
                result["%" + "text"] += str(element.tail).strip()
            if((not(result.has_key("%" + "xmlns")))):
                result["%" + "xmlns'] = str()
            if((type(xml.QName(element.tag).namespace) != type(None))):
                result["%" + "xmlns"] += str(xml.QName(element.tag).namespace).strip()
        else:
            if((not(result.has_key("%" + "name")))):
                result["%" + "name"] = str()
            if((type(xml.QName(element[0]).localname) != type(None))):
                result["%" + "name"] += str(xml.QName(element[0]).localname).strip()
            if((not(result.has_key("%" + "text")))):
                result["%" + "text"] = str()
            if((type(element[1]) != type(None))):
                result["%" + "text"] += str(element[1]).strip()
            if((type(element[1]) != type(None))):
                result["%" + "text"] += str("        ").strip()
            if((not(result.has_key("%" + "xmlns")))):
                result["%" + "xmlns"] = str()
            if((type(xml.QName(element[0]).namespace) != type(None))):
                result["%" + "xmlns"] += str(xml.QName(element[0]).namespace).strip()
    if((type(element) != type(tuple()))):
        for item in element.getchildren():
            if((not(result.has_key("#" + xml.QName(item.tag).text)))):
                result["#" + xml.QName(item.tag).text] = list()
            if((type(item) != type(None))):
                result["#" + xml.QName(item.tag).text].append(parse(item))

    return(result)


def main():
    print(parse(xml.fromstring(open("d:/export.xml", "rb").read())))

    return


if((__name__ == "__main__")):
    main()
